content/docs.pl.md

```diff
@@ -0,0 +1,184 @@
+
+
+- **Techniki wystarczające** są przykładami sposobów spełnienia kryteriów sukcesu. Jeśli zastosujesz techniki wystarczające, spełnisz kryteria sukcesu.
+- **Techniki pomocnicze** to sugerowane sposoby poprawy dostępności. Są one bardzo pomocne dla niektórych użytkowników i czasem stanowią jedyny sposób, w jaki niektórzy użytkownicy mogą uzyskać dostęp do niektórych typów treści.
+- **Błędy** to rzeczy, które powodują bariery dostępności i nie spełniają określonych kryteriów sukcesu.
+
+Więcej informacji:
+- **[O technikach](https://www.w3.org/WAI/WCAG22/Understanding/understanding-techniques)**
+- **[Lista technik dla WCAG 2.2](https://www.w3.org/WAI/WCAG22/Techniques/)**
+- [Lista technik dla WCAG 2.1](https://www.w3.org/WAI/WCAG21/Techniques/)
+- [Lista technik dla WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/)
+
+### Zasady testowania zgodności (ACT)
+
+<cite>Zasady testowania zgodności (ACT)</cite> opisują sposoby sprawdzania zgodności z kryteriami sukcesu WCAG. Ich zadaniem jest przede wszystkim opracowywanie narzędzi ewaluacyjnych i metodologii testów. (Są one bardziej niezawodne niż testy przedstawiane w omówieniach technik.)
+
+* [O zasadach testowania zgodności (ACT) WCAG 2](/standards-guidelines/act/rules/about/)
+* [Lista wszystkich zasad testowania zgodności (ACT) WCAG 2](/standards-guidelines/act/rules/)
+
+## Dodatkowe wytyczne {#supplement}
+
+Dodatkowe wytyczne wykraczają poza wymagania WCAG. **Nie są wymagane**, aby spełnić wymagania WCAG. Ale zastosowanie wielu z tych wskazówek jest niezbędne, aby zapewnić dostępność ososobom z pewnymi zaburzeniami, w tym z trudnościami poznawczymi i trudnościami w uczeniu się.
+
+* **[O dodatkowych wytycznych](/WCAG2/supplemental/about/)**
+* **[Lista wszystkich dodatkowych wytycznych](/WCAG2/supplemental/)**
+
+## Dodatkowe materiały
+
+**Zanim zaczniesz pracować z WCAG**, prawdopodobnie chcesz przeczytać te materiały:
+* [[Wprowadzenie do dostępności internetowej]](/fundamentals/accessibility-intro/)
+* [[Zasady dostępności]](/fundamentals/accessibility-principles/)
+* [[Omówienie Wytycznych dla dostępności internetowej (WCAG)]](/standards-guidelines/wcag/)
+
+Inne materiały dotyczą konkretnych tematów, na przykład:
+* [Samouczki dotyczące dostępności internetowej](https://www.w3.org/WAI/tutorials/) omawiają strukturę strony, menu, obrazy, tabele, karuzele i formularze
+* [Tworzenie dostępnych mediów audio i wideo](/media/av/) dotyczą multimediów
+
+Zachęcamy Cię do rozejrzenia się po witrynie W3C WAI, aby znaleźć inne informacje, które być może zechcesz przeczytać lub podzielić się nimi z innymi.
+
+
+{% include_cached excol.html type="start" id="diagram" %}
+
+## Stary diagram
+
+{% include_cached excol.html type="middle" %}
+
+Diagram poniżej przedstawia różne dokumenty WCAG 2 przeznaczone do różnych celów. Wykorzystuje „WCAG 2.0”, ale nadal ma zastosowanie do WCAG 2.1 i WCAG 2.2.
+
+{% include image.html src="wcag20docs-pl.svg" alt="" %}
```

Comment:
@zwiastunsw Thanks for suggesting translation of the diagram. 

However, this diagram needs a major update. This will probably not happen soon though. When done, we will provide the source file and instructions to translate it. In the meantime, please keep the original diagram in English.